upgrade/satellite.py

```diff
@@ -131,10 +131,7 @@ def satellite_upgrade(zstream=False):
             )
         foreman_maintain_package_update()
 
-    if bz_bug_is_open(1995650) and settings.upgrade.to_version == '6.10':
-        run("yum remove -y rubygem-passenger")
-
-    if settings.upgrade.to_version == '6.10':
+    if settings.upgrade.from_version == '6.9':
```

Comment:
@lpramuk It will get fail if we ran the upgrade for z-stream, I mean from 6.9 to 6.9.z.
`if settings.upgrade.to_version == '6.10' and settings.upgrade.from_version == "6.9":`